docs/syzbot.md

```diff
@@ -164,8 +164,20 @@ so please treat the results with understanding. You may consult the provided
 Suggestions and patches that improve bisection quality for common cases are
 [welcome](https://github.com/google/syzkaller/issues/1051).
 
+Syzbot supports cause bisection(find the commit that introduces a bug) and fix bisection(find the commit that fixes a bug).
+
+The web UI for a specific kernel(say [upstream linux](https://syzkaller.appspot.com/upstream)) shows the `Bisected` status for all bugs.
+
 <div id="bisection"/>
 
+### Fix Bisection
```

Comment:
We don't capitalize second word in headers anywhere in this doc, please keep the style consistent.